package.json

```diff
@@ -25,6 +25,7 @@
     "dev:packages": "yarn tsc --build --watch --preserveWatchOutput tsconfig.packages.json",
     "tenderly": "yarn tenderly:spawn && yarn env e2e && yarn concurrently --kill-others -n \"rpc-proxy,unchained-proxy\" \"yarn tenderly:rpc-proxy\" \"yarn tenderly:unchained-proxy\"",
     "env": "yarn ts-node --skipProject ./scripts/bootstrap-env.ts",
+    "generate:caip-adapters": "npm --prefix ./packages/caip run generate",
```

Comment:
Not a mistake, tried many many hoops to get this running with `yarn` with no success 😥

Reply:
below suggestion should work by leveraging yarn workspace directive